# HG changeset patch
# User jwe
# Date 776808677 0
#      Sat Aug 13 20:11:17 1994 +0000
# Node ID 91ab3cfc23763010218f671e778d06728ae1d016
# Parent  4f65175911a60b46deb141df05144743adbbcc9b
[project @ 1994-08-13 20:11:17 by jwe]
Initial revision

diff --git a/src/load-save.h b/src/load-save.h
new file mode 100644
--- /dev/null
+++ b/src/load-save.h
@@ -0,0 +1,44 @@
+// load-save.h                                              -*- C++ -*-
+/*
+
+Copyright (C) 1992, 1993, 1994 John W. Eaton
+
+This file is part of Octave.
+
+Octave is free software; you can redistribute it and/or modify it
+under the terms of the GNU General Public License as published by the
+Free Software Foundation; either version 2, or (at your option) any
+later version.
+
+Octave is distributed in the hope that it will be useful, but WITHOUT
+ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+for more details.
+
+You should have received a copy of the GNU General Public License
+along with Octave; see the file COPYING.  If not, write to the Free
+Software Foundation, 675 Mass Ave, Cambridge, MA 02139, USA.
+
+*/
+
+#if !defined (octave_load_save_h)
+#define octave_load_save_h 1
+
+class tree_constant;
+class ostream;
+
+extern int save_ascii_data (ostream& os, const tree_constant& t,
+			    char *name = 0, int mark_as_global = 0,
+			    int precision = 0);
+
+extern int save_three_d (ostream& os, const tree_constant& t,
+			 int parametric = 0);
+
+#endif
+
+/*
+;;; Local Variables: ***
+;;; mode: C++ ***
+;;; page-delimiter: "^/\\*" ***
+;;; End: ***
+*/
